packages/opc-nav/src/opc-nav.styles.ts

```diff
@@ -0,0 +1,156 @@
+import { css } from "lit-element";
```

Comment:
@akhilmhdh any reason why we are using css when we can use .scss?

Reply:
Well one reason was from documentation, lit even suggested to use there css in js solution of lit library for some optimization and performance. And yes i was using a css-in-js style in the shared elements package.

Reply:
I would suggest you to use the .scss way as you would as be importing and using opc-styles for different sizes

Reply:
Ok will make that change

Reply:
Cool